starter-code/apple-pie-starter-code/css/styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300, 400, 500, 600');
+
+body {
+	font-family: 'Roboto';
+}
+
+li, ul, ol{
+	margin: 0;
+	padding: 0;
+}
+
+h1 {
+	color: white;
+	font-size: 80px;
+	font-weight: 400;
+	margin: auto auto;
+	vertical-align: center;
+	
+}
+
+h2 {
+	text-align:left;
+	font-weight: 400;
+	font-size: 1.3em;
+}
+
+p {
+	font-size: 1.3em;
+	font-weight: 200;
+}
+
+hr {
+	border-top: solid 2px rgba(0, 0, 0, 0.4);
+	margin: 0 15% 40px 15%;
+}
+
+.header {
+	text-align: center;
+	margin: 0;
+	background-image: url("../images/apple-pie.jpg");
+	background-size: cover;
+	height: 600px;
+	width: 100%;
+	margin: 0 5%;
```

Comment:
You've used margin twice here - by removing 'margin: 0 5%' you fix the problem with the margin on the left. 

if you add 
```
body {
margin: 0
}
```
it will remove the margin from the entire page and then the header will take up the full space